distribution/tools/plugin-cli/src/main/java/org/elasticsearch/plugins/InstallPluginCommand.java

```diff
@@ -605,11 +707,21 @@ private PluginInfo loadPluginInfo(Terminal terminal, Path pluginRoot, Environmen
         return info;
     }
 
+    private static final String LIB_TOOLS_PLUGIN_CLI_CLASSPATH_JAR;
+
+    static {
+        LIB_TOOLS_PLUGIN_CLI_CLASSPATH_JAR =
```

Comment:
nit: shouldn't this be jars, plural (ie it matches many jars, not just one)?

Reply:
It is the pattern for matching *a* JAR under the lib/tools/plugin-cli path, it is only that there can be multiple matches.